native-heapdump-example/README.md

```diff
@@ -0,0 +1,130 @@
+# Create a Heap Dump from a Native Executable
+
+You can create a heap dump of a running executable to monitor its execution. Just like any other Java heap dump, it can be opened with the [VisualVM](https://www.graalvm.org/latest/tools/visualvm/) tool. 
+
+For this, you need to add `jvmstat` to the `--enable-monitoring` option (for example, `--enable-monitoring=heapdump,jvmstat`).
+This will allow VisualVM to pick up and list running Native Image processes.
+You can then request heap dumps in the same way you can request them when your application runs on the JVM (for example, right-click on the process, then select "Heap Dump").
```

Comment:
Update this paragraph like this:

_You can then request heap dumps in the same way you can request them when your application runs on the JVM (for example, right-click on the process, then select "Heap Dump"). There are different ways to create heap dumps:_